### PR TITLE
Hotfix: Fix inverted anonymous boolean

### PR DESCRIPTION
**Pull request by CarlosQ96**
We currently have this boolean inverted, Ramin seems to use this on the new design. So fixing it once and for all 👍

### CODE REVIEW

**Comment by CarlosQ96** [comment]
@mohammadranjbarz Validated with Mateo that they are not using this boolean in the front.
Seems that Ramin will do in the new design, so fixing it now.

Checked the rest of the segment events and it seems they are using it correctly so nothing to change with segment.

File: migration/1638386837934-FixAnonymousDonationBoolean.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    );
+    await queryRunner.query(
+      'UPDATE donation SET anonymous = false WHERE "userId" IS NOT NULL',
+    );
+  }
+
+  // Revert in case something happens, seems only the new design will use this
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'UPDATE "donation" SET anonymous = false WHERE "userId" IS NULL',
```
Comment (mohammadranjbarz) [inline migration/1638386837934-FixAnonymousDonationBoolean.ts:18]: @CarlosQ96  I think you should set anonymous `true` in this case

File: migration/1638386837934-FixAnonymousDonationBoolean.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    );
+    await queryRunner.query(
+      'UPDATE "donation" SET anonymous = true WHERE "userId" IS NOT NULL',
```
Comment (mohammadranjbarz) [inline migration/1638386837934-FixAnonymousDonationBoolean.ts:21]: @CarlosQ96  I think you should set anonymous `false` in this case

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]

File: migration/1638386837934-FixAnonymousDonationBoolean.ts (hunk 1 above)
Comment (CarlosQ96) [reply migration/1638386837934-FixAnonymousDonationBoolean.ts:18]: I was thinking what would be the down method in this case? I was like reverting it to how it was  before. The up method is like you are refering.

Comment (mohammadranjbarz) [reply migration/1638386837934-FixAnonymousDonationBoolean.ts:18]: Oh man sorry , my mistake
You are totally right

**Review by mohammadranjbarz — APPROVED** [review]
Seems good to me

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @CarlosQ96 
LGTM

**Pull request merged**